src/d3a/models/strategy/load_hours.py

```diff
@@ -300,10 +301,9 @@ def _double_sided_market_event_tick(self, market):
     def event_tick(self):
         """Post bids on market tick. This method is triggered by the TICK event."""
         for market in self.active_markets:
-            if ConstSettings.IAASettings.MARKET_TYPE == 1:
+            if ConstSettings.IAASettings.MARKET_TYPE == SpotMarketTypeEnum.ONE_SIDED.value:
                 self._one_sided_market_event_tick(market)
-            elif ConstSettings.IAASettings.MARKET_TYPE == 2 or \
-                    ConstSettings.IAASettings.MARKET_TYPE == 3:
+            elif ConstSettings.IAASettings.MARKET_TYPE == SpotMarketTypeEnum.TWO_SIDED.value:
```

Comment:
Great. Can you please perform this refactoring also for all other cases where we compare to raw integers (i just saw that there are a lot). Thanks a lot!